recipe/conda_build_config.yaml

```diff
@@ -5,8 +5,13 @@ major_minor_ver:
   - 10.2
   - 11.0
   - 11.1
+  - 11.2
 
-cuda_compiler_version:                  # [linux64]
-  - None                                # [linux64]
-docker_image:                           # [linux64]
-  - condaforge/linux-anvil-cos7-x86_64  # [linux64]
+cuda_compiler_version:                          # [linux64]
+  - None                                        # [linux64]
+docker_image:                                   # [linux64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64  # [linux64]
+cudnn:                                          # [linux64]
+  - undefined                                   # [linux64]
+cdt_name:                                       # [linux64]
+  - cos7                                        # [linux64]
```

Comment:
I have a question: Would this setup leak to downstream packages? CUDA 9.x/10.x work fine on CentOS 6, so if a leak could happen we should use the image `linux-anvil-x86_64` (without the `cos7` decorator). (This question is mainly about consistency in sysroot/glibc versions.)

Reply:
I guess it's here to avoid the manual migration and hence the egg-and-chicken problem, but I don't actually know if it's necessary. I can also try providing all the zipped keys and see if that's better?

Reply:
Given it's just binary repackaging of the libraries I don't think this would cause any leakage.

Reply:
Good to know, thanks Keith.